Content/default/.editorconfig

```diff
@@ -5,8 +5,7 @@ indent_style = space
 indent_size = 4
 charset = utf-8
 trim_trailing_whitespace = true
-insert_final_newline = false
+insert_final_newline = true
 
-[src/Client/*.fs]
-# Feliz style
-fsharp_single_argument_web_mode=true
\ No newline at end of file
+fsharp_multiline_block_brackets_on_same_column = true
```

Comment:
should we use these settings for all files, or just `src/Client/*.fs` ?
I'm getting an impression that they are specifically for the purpose of Fable/Feliz

Reply:
I would vote for consistent rules between server/client/shared.